editor/project_manager.cpp

```diff
@@ -250,8 +250,12 @@ class ProjectDialog : public ConfirmationDialog {
 
 	void _create_folder() {
 
-		if (project_name->get_text() == "" || created_folder_path != "")
+		if (project_name->get_text() == "" || created_folder_path != "" || project_name->get_text().ends_with(".") || project_name->get_text().ends_with(" "))
+		{
```

Comment:
As per our style guide the bracket should be on the same line as the `if`.

Reply:
I'll fix it later today.

Reply:
Sorry for the delay. Fixed now with commit#164dcea